pkg/hintrunner/zero/zerohint_ec_test.go

```diff
@@ -1254,6 +1254,23 @@ func TestZeroHintEc(t *testing.T) {
 				}),
 			},
 		},
+		"GetHighLen": {
+			{
+				operanders: []*hintOperander{
+					{Name: "len_hi", Kind: uninitialized},
+					{Name: "scalar_u.d0", Kind: apRelative, Value: feltString("123")},
+					{Name: "scalar_u.d1", Kind: apRelative, Value: feltString("456")},
+					{Name: "scalar_u.d2", Kind: apRelative, Value: feltString("789")},
+					{Name: "scalar_v.d0", Kind: apRelative, Value: feltString("987")},
+					{Name: "scalar_v.d1", Kind: apRelative, Value: feltString("654")},
+					{Name: "scalar_v.d2", Kind: apRelative, Value: feltString("321")},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newGetHighLenHint(ctx.operanders["len_hi"], ctx.operanders["scalar_u.d0"], ctx.operanders["scalar_v.d0"])
+				},
+				check: varValueEquals("len_hi", feltInt64(250)),
```

Comment:
unit tests are not supposed to be built only to pass.
You are supposed to think of the expected value, and then test your code.
Ideally, you would run the cairo file on the python VM, add print statements to see the output  on the python vm of the computation of the hint you want to implement , which will be your expected value for your unit test

What you did is running your hint and writting in the expected value the value returned by your code. This is not a good practice at all, just by thinking a few seconds it doesnt make any sense to expect `250` bits from either D2 (value: 321) or V2 (value: 789)